src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Nullable.cs

```diff
@@ -222,8 +220,6 @@ public class Attribute
                 // error CS1729: 'Attribute' does not contain a constructor that takes 0 arguments
                 Diagnostic(ErrorCode.ERR_BadCtorArgCount).WithArguments("System.Attribute", "0").WithLocation(1, 1),
```

Comment:
> Diagnostic(ErrorCode.ERR_BadCtorArgCount).WithArguments("System.Attribute", "0").WithLocation(1, 1), [](start = 15, length = 101)

Do we understand why we report less errors now for types and methods?

Reply:
One of those was produced when we tried to synthesize a method body using the `Attribute` constructor, but the `Attribute` type itself is missing.
Now, we bail out early if the type is missing (we produce an empty/dummy body in such case). See `InjectedNonNullTypesAttributeSymbol.MakeAttributes`.

---
In reply to: [215497415](https://github.com/dotnet/roslyn/pull/29180#discussion_r215497415) [](ancestors = 215497415)